### DISCUSSION TITLE
Anyway to show a custom modal containing a form when editing?

### DISCUSSION THREAD

**Discussion by msueping** [KevinVandy/material-react-table#647] [Q&A] [answered]
I'm aware of the various edit modes when using this library. I'd like to have my own MUI dialog show instead of what happens currently when the editingMode is set to "modal".

Is this possible and if so, how do I make that happen?

TIA

**Comment by KevinVandy** [comment, marked as answer]
A "custom" edit mode is coming in v2. It handles the state, but not the UI so that you can 100% control your own UI for editing.

**Reply by msueping** [reply]
Thank you Kevin for the prompt reply. That sounds promising.
